resources/views/listing/partials/filter/category.blade.php

```diff
@@ -10,9 +10,7 @@
     <div slot-scope="{ aggregations, setQuery, value }">
         <category-filter :aggregations="aggregations" :value="value" :set-query="setQuery">
             <div slot-scope="{ hasResults, results }">
-                <strong v-if="hasResults">@lang('Category')</strong>
-                <input class="hidden" type="radio" name="category" value="" :checked="!value" v-on:change="setQuery({})"/>
```

Comment:
This might hinder functionality if removed. I suggest leaving it in.

As for the category title; every other filter has a title, i think the default should be to have that title. With the option to remove it